selftests/checkall

```diff
@@ -80,12 +80,12 @@ results_dir_content() {
 
 [ "$SKIP_RESULTSDIR_CHECK" ] || RESULTS_DIR_CONTENT="$(ls $RESULTS_DIR 2> /dev/null)"
 if [ "$TRAVIS" == "true" ]; then
-    run_rc lint 'RESULT=$(mktemp); inspekt --exclude=.git lint &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
+    run_rc lint 'RESULT=$(mktemp); inspekt lint --exclude=.git &>$RESULT || { cat $RESULT; rm $RESULT; exit -1; }; rm $RESULT'
```

Comment:
To those still actively using projects with old syntax and new syntax, this is a very ugly modification to the new inspector to be able to run them without any issues:

```diff
diff --git a/inspektor/cli/app.py b/inspektor/cli/app.py
index bdfc939..0238f04 100644
--- a/inspektor/cli/app.py
+++ b/inspektor/cli/app.py
@@ -17,10 +17,18 @@ Implements the base inspektor application.
 """
 import sys
 
+i = 0
+for i, arg in enumerate(sys.argv[1:]):
+    if arg.startswith("--"):
+        continue
+    break
+if i:
+    sys.argv = [sys.argv[0]] + sys.argv[i+1:] + sys.argv[1:i+1]
+print " ".join(sys.argv)
+
 from cliff.app import App
 from cliff.commandmanager import CommandManager
 
-
 class InspektorApp(App):
 
     def __init__(self):
```